app/src/main/res/layout/fragment_favorite.xml

```diff
@@ -6,6 +6,19 @@
     <FrameLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent">
+       <LinearLayout
+           android:layout_width="match_parent"
+           android:layout_height="wrap_content"
+           android:orientation="vertical">
+           <TextView
```

Comment:
The text would look better IMO if it is both horizontally and vertically centred.

Reply:
Okay, I will resolve this

Reply:
screenshot updated